src/main/java/com/mathworks/ci/RunMatlabBuildBuilder.java

```diff
@@ -108,10 +119,10 @@ private int execMatlabCommand(FilePath workspace, Launcher launcher,
         // Create MATLAB script
         createMatlabScriptByName(uniqeTmpFolderPath, uniqueBuildFile, workspace, listener, envVars);
         ProcStarter matlabLauncher;
-
+        String options = getStartupOptions() == null ? "" : getStartupOptions().getStartupOptions();
         try {
             matlabLauncher = getProcessToRunMatlabCommand(workspace, launcher, listener, envVars,
-                    "cd('"+ uniqeTmpFolderPath.getRemote().replaceAll("'", "''") +"');"+ uniqueBuildFile, uniqueTmpFldrName);
+                    "cd('"+ uniqeTmpFolderPath.getRemote().replaceAll("'", "''") +"');"+ uniqueBuildFile, options, uniqueTmpFldrName);
```

Comment:
Do we pass "" here if startupOptions is NULL  like uniqueFile,,  curious to see the log output when no options set